big_tests/dynamic_domains.config

```diff
@@ -1,11 +1,21 @@
 %% Options defined here are used when testing dynamic domains, see 'dynamic_domains.spec'
 %% They take precedence over 'test.config'
 
+{disable_disco_tests, true}.
+{disable_s2s_tests, true}.  %% Requires mod_register
+{disable_carbons_tests, true}.
+{disable_roster_tests, true}.
+
 {hosts, [{mim,  [{node, mongooseim@localhost},
                  {domain, <<"domain.example.com">>},
+                 {host_type, <<"test type">>},
                  {secondary_domain, <<"domain.example.org">>},
+                 {secondary_host_type, <<"test type">>},
```

Comment:
Why's this the same than the first host type?

Reply:
idk, it works like that xdd
I think it's a value, you get if you try to convert "domain.example.org" to host type.